### PR TITLE
Send permit update emails to the submission email

### PR DESCRIPTION
**Pull request by LaurelOlson**
## Description

Instead of emailing each reviewer in the jurisidction, just send a single notification to the configured submission email for the permit type and jurisdiction.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents

[BPHH-1398](https://hous-bssb.atlassian.net/browse/BPHH-1398)

## UI Accessibility Checklist

_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

- [ ] Markup uses semantic HTML?
- [ ] Additional context added through `aria-roles` and `aria-labels`?
- [ ] Checked
      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?
- [ ] Usable with a keyboard and navigable in a logical order?
- [ ] Usable using a screen reader (e.g. Voiceover for Mac) with enough context for the user?

## General Checklist

- [ ] ✅ Provide tests for your changes where relevant
- [x] 📝 Use descriptive commit messages
- [ ] 📗 Update any related documentation and include any relevant screenshots
- [ ] 📱 For UI-related tasks, ensure responsive design is implemented across multiple screen size
- [x] 🗂️ Move your relevant story/task to the _Ready for Code Review_ state

## [optional] Are there any post-deployment tasks we need to perform?

n/a

### CODE REVIEW

File: app/models/permit_application.rb [hunk 1]
```diff
@@ -136,9 +136,13 @@ def permit_type_and_activity
...
   end
 
+  def permit_type_submission_contact
+    jurisdiction.permit_type_submission_contacts.where(permit_type: permit_type)
+  end
+
   def send_submit_notifications
     PermitHubMailer.notify_submitter_application_submitted(submitter, self).deliver_later
-    jurisdiction.users.each { |user| PermitHubMailer.notify_reviewer_application_received(user, self).deliver_later }
+    PermitHubMailer.notify_reviewer_application_received(permit_type_submission_contact, self).deliver_later
```
Comment (taw27) [inline app/models/permit_application.rb:145]: `permit_type_submission_contact` isn't this an array?

Comment (LaurelOlson) [reply app/models/permit_application.rb:145]: The jurisdiction has a has_many but I added the above singular method to get the contact for that specific permit based on the jurisdiction AND the permit type

Comment (LaurelOlson) [reply app/models/permit_application.rb:145]: Ah yes thank you, will change to a find_by

**Review by taw27 — APPROVED** [review]

File: app/jobs/remind_reviewer_job.rb [hunk 2]
```diff
@@ -6,7 +6,10 @@ def perform()
       unviewed_applications = jur.unviewed_permit_applications
 
       if unviewed_applications.any?
-        jur.users.each { |user| PermitHubMailer.remind_reviewer(user, unviewed_applications).deliver }
+        jur.permit_type_submission_contacts.each do |contact|
+          apps = unviewed_applications.where(permit_type: contact.permit_type)
```
Comment (taw27) [inline app/jobs/remind_reviewer_job.rb:10]: array of contact here too right?

**Review by taw27 — CHANGES_REQUESTED** [review]

File: app/jobs/remind_reviewer_job.rb (hunk 2 above)
Comment (LaurelOlson) [reply app/jobs/remind_reviewer_job.rb:10]: this one is expected to be an array

**Review by taw27 — APPROVED** [review]

**Pull request merged**